tests/integration/schema/simple_test.go

```diff
@@ -78,7 +78,7 @@ func TestSchemaSimpleErrorsGivenDuplicateSchema(t *testing.T) {
 				Schema: `
 					type Users {}
 				`,
-				ExpectedError: "schema type already exists",
+				ExpectedError: "collection already exists",
```

Comment:
note: These changed as the order in which `loadSchema` was called changed slightly.  I think the new error is more correct anyway.